app/src/test/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorViewTest.kt

```diff
@@ -155,7 +164,44 @@ class CreditCardEditorViewTest {
     }
 
     @Test
-    fun `GIVEN a credit card WHEN the save button is clicked THEN interactor is called`() {
+    fun `GIVEN the credit input are filled with valid credit card number WHEN the save button is clicked THEN interactor is called`() {
```

Comment:
I can't help myself from simplifying this since you wrote `GIVEN a valid credit card` in the next test.